### PR TITLE
Part 1: Why we decided to do SCB v2

### PR DESCRIPTION
**Pull request by Weltraumschaf**
Article about the architectureofv1 and why we decided to make v2.

### CODE REVIEW

File: blog/2020-11-04-why-securecodebox-version-2.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+- use k8s to schedule scans as custom resources in jobs
+
+### Design Goals
+
+What about the design goals from the *version 1* architecture? Let's go through each of them:
+
+>  It should be possible to easily integrate new scanners.
+
+- Container as in v1
+- but way simpler image because CLI args andparsing stuff is outside
```
Comment (rseedorff) [inline blog/2020-11-04-why-securecodebox-version-2.md:120]: typo "and parsing"

File: blog/2020-11-04-why-securecodebox-version-2.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+- args must be provided "from outside"
+- lurcher sidecar to collect data foranykind of scanner
```
Comment (rseedorff) [inline blog/2020-11-04-why-securecodebox-version-2.md:122]: sidecar (lurcher) to collect data for any kind of scanner

File: blog/2020-11-04-why-securecodebox-version-2.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+- simpler parser hooks as k8s jobs
+
+>  All components should be loosly coupled to easily swap them.
+
+- same decoupling as in version 1
+- if a scanner changes only the according parser must be changed
+- but total hard-coupling  to k8s
```
Comment (rseedorff) [inline blog/2020-11-04-why-securecodebox-version-2.md:129]: typo (spacing)

File: blog/2020-11-04-why-securecodebox-version-2.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+> The definition and implementation of a scan process should be easy.
+
+- own YAML resource syntax
+- way easier than generating a BPMN model in Camunda
+- it's just writing some YAML
+
+New Design goals:
+
+- scanner container does not run all the time idling
+- easy use of scanners in  cloud based projects
```
Comment (rseedorff) [inline blog/2020-11-04-why-securecodebox-version-2.md:146]: typo (spacing)

**Review by rseedorff — CHANGES_REQUESTED** [review]

**Comment by Weltraumschaf** [comment]
I messed up with my branches. Will open new PR.

**Pull request closed without merging**